### DISCUSSION TITLE
always-auth for uplinks

### DISCUSSION THREAD

**Discussion by sergeimonakhov** [verdaccio/verdaccio#2284] [Ideas]
**Is your feature request related to a problem?**
Hi, I have 2 verdaccio servers and I wanted to proxy (add to uplinks) from one to the other, but since `$authenticated` is used for packages access, the request must contain authentication headers, but this is not the [case](https://github.com/verdaccio/verdaccio/blob/38ca095fb041c768c1ea85f4d922f808c124777c/src/lib/up-storage.ts#L464). I get package.json, but download .tgz responsed 401 error

**Describe the solution you'd like**
Add `always-auth: true/false` key to uplinks config

**Describe alternatives you've considered**

**Additional context**

**Comment by spovich** [comment]
Hi, I'm experiencing the same issue with an authenticated uplink. The server successfully gets the package metadata, but the subsequent request for the tarball returns a 401. @sergeimonakhov did you ever find a way around this?
